engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-products.controller.js

```diff
@@ -6,41 +6,92 @@
  * @requires translate
  * @requires ContentHost
  * @requires Product
- * @requires CurrentOrganization
  *
  * @description
  *   Provides the functionality for the content-host products action pane.
  */
 angular.module('Bastion.content-hosts').controller('ContentHostProductsController',
-    ['$scope', 'translate', 'ContentHost', 'Product', 'CurrentOrganization',
-    function ($scope, translate, ContentHost, Product, CurrentOrganization) {
+    ['$scope', 'translate', 'HostSubscription',
+    function ($scope, translate, HostSubscription) {
+        var defaultOverride = "default";
 
         $scope.successMessages = [];
         $scope.errorMessages = [];
         $scope.displayArea = { working: true, isAvailableContent: false };
 
-        $scope.isAnyAvailableContent = function (products) {
-            var isAvailableContent = false;
-            angular.forEach(products, function (product) {
-                if (product['available_content'].length > 0) {
-                    isAvailableContent = true;
-                }
-            });
-            return isAvailableContent;
+        $scope.getEnabledText = function (enabled, overrideEnabled) {
+            var enabledText;
+            overrideEnabled = overrideEnabled + "";
```

Comment:
it comes back from the server as a string, but doesn't matter what we send to it.  I figured it simplified things to handle both cases.